lumigator/infra/mzai/helm/lumigator/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: ""
```

Comment:
We should fill this in with the tag in the repo, right?

Reply:
Yep, we should automatize the process when we create a new release, in order to change the appVersion here, tag the image in DockerHub, etc. I created this [story](https://mzai.atlassian.net/browse/LUMI-51).